### PR TITLE
Add message size with tooltip, use variable polling duration

### PR DESCRIPTION
**Pull request by MikeEdgar**
Added "size" column with tooltip:

image

### CODE REVIEW

**Comment by MikeEdgar** [comment]
I put the tooltip in the i18n message file, but maybe the way to extract it for the table header is not the best. @riccardo-forina , let me know if you have any suggestions on that.

File: ui/app/[locale]/kafka/[kafkaId]/topics/[topicId]/messages/_components/MessageDetails.tsx [hunk 1]
```diff
@@ -20,7 +21,9 @@ import {
   Text,
   TextContent,
   TextVariants,
+  Tooltip,
 } from "@/libs/patternfly/react-core";
+import { HelpIcon } from "@patternfly/react-icons";
```
Comment (riccardo-forina) [inline ui/app/[locale]/kafka/[kafkaId]/topics/[topicId]/messages/_components/MessageDetails.tsx:26]:
```suggestion
import { HelpIcon } from "@/libs/patternfly/react-icons";
```

**Review by riccardo-forina — APPROVED** [review]
LGTM!

**Comment by shirimordechay** [comment]
The tooltip text might change but other than that LGTM

**Pull request merged**